e2etests/specs/example.spec

```diff
@@ -1,11 +1,6 @@
-# Getting Started with Gauge
+# Agile Pathway spec
 
-This is an executable specification file. This file follows markdown syntax. Every heading in this file denotes a scenario. Every bulleted point denotes a step.
-To execute this specification, use
-	npm test
+## Home page
```

Comment:
Just one simple test in this pull request, more will follow in future pull requests.